source/editor/lib/DocumentationMapping.json

```diff
@@ -1,41 +1,48 @@
+  "infotrigger": "/develop/manual/config/widgets/infotrigger/index.html#infotrigger", 
+  "line": "/develop/manual/config/widgets/line/index.html#line", 
+  "link": "/develop/manual/config/widgets/plugins/link/index.html#link", 
+  "mapping": "/develop/manual/config/mapping.html#mapping", 
+  "multitrigger": "/develop/manual/config/widgets/multitrigger/index.html#multitrigger", 
+  "notificationcenterbadge": "/develop/manual/config/widgets/notificationcenterbadge/index.html#notificationcenterbadge", 
+  "page": "/develop/manual/config/widgets/page/index.html#page", 
+  "pagejump": "/develop/manual/config/widgets/pagejump/index.html#pagejump", 
+  "powerspectrum": "/develop/manual/config/widgets/plugins/powerspectrum/index.html#powerspectrum", 
+  "pushbutton": "/develop/manual/config/widgets/pushbutton/index.html#pushbutton", 
+  "refresh": "/develop/manual/config/widgets/refresh/index.html#refresh", 
+  "reload": "/develop/manual/config/widgets/reload/index.html#reload", 
+  "rgb": "/develop/manual/config/widgets/rgb/index.html#rgb", 
+  "roundbar": "/develop/manual/config/widgets/roundbar/index.html#roundbar", 
+  "slide": "/develop/manual/config/widgets/slide/index.html#slide", 
+  "strftime": "/develop/manual/config/widgets/plugins/strftime/index.html#strftime", 
+  "switch": "/develop/manual/config/widgets/switch/index.html#switch", 
+  "system-voraussetzungen": "/develop/manual/#system-voraussetzungen", 
+  "text": "/develop/manual/config/widgets/text/index.html#text", 
+  "toggle": "/develop/manual/config/widgets/toggle/index.html#toggle", 
+  "tr064": "/develop/manual/config/widgets/plugins/tr064/index.html#tr064", 
+  "trigger": "/develop/manual/config/widgets/trigger/index.html#trigger", 
+  "urltrigger": "/develop/manual/config/widgets/urltrigger/index.html#urltrigger", 
+  "video": "/develop/manual/config/widgets/video/index.html#video", 
+  "visu-config-details": "/develop/manual/config/index.html#visu-config-details", 
+  "web": "/develop/manual/config/widgets/web/index.html#web", 
+  "widgets": "/develop/manual/config/widgets/index.html#widgets", 
+  "xml-format": "/develop/manual/config/xml-format.html#xml-format", 
+  "xml-format-files": "/develop/manual/config/xml-format.html#xml-format-files", 
+  "xml-format-header": "/develop/manual/config/xml-format.html#xml-format-header", 
+  "xml-format-icons": "/develop/manual/config/xml-format.html#xml-format-icons", 
+  "xml-format-mappings": "/develop/manual/config/xml-format.html#xml-format-mappings", 
+  "xml-format-pages": "/develop/manual/config/xml-format.html#xml-format-pages", 
+  "xml-format-plugins": "/develop/manual/config/xml-format.html#xml-format-plugins", 
+  "xml-format-statusbar": "/develop/manual/config/xml-format.html#xml-format-statusbar", 
+  "xml-format-stylings": "/develop/manual/config/xml-format.html#xml-format-stylings", 
+  "xml-format-templates": "/develop/manual/config/xml-format.html#xml-format-templates"
```

Comment:
Please remove the changes for this file from the PR. The file is generated when you build the docs, but the change must not be commited.

Reply:
I have no experience with that. Can you support this? I have commited all changes in VS Code.